src/format/CsvExporter.cpp

```diff
@@ -79,7 +79,7 @@ QString CsvExporter::exportGroup(const Group* group, QString groupPath)
     if (!groupPath.isEmpty()) {
         groupPath.append("/");
     }
-    groupPath.append(group->name());
+    groupPath.append(group->name().replace("/", "\\"));
```

Comment:
Shouldn't this be `\/`?

Reply:
I think it's not about escaping the character, because '/' is a valid character instead of for example '\' which needs to be escaped. 
The problem there is to distinguish between a slash which belongs to the path and a slash which is part of the group name. 
To parse the string the / in the group name needs to be replaced in the csv file somehow. I thought about doing this with '\\'.

Reply:
Yes you are right. 
One proper way to fix that problem would be to store the group path and group name as separate column in the csv file. 
Then the group name can be stored without replacing it with something.

Reply:
Ok. So finally you think we should accept this behavior and keep it as it is?

Reply:
No, changing the slash direction makes no sense, plus \\ is more likely to appear than \/. The other alternative is to use a placeholder like {SLASH}, but that is plain ugly and wrong.

Reply:
I didn't mean to merge my pull request. 
I meant to just keep the behavior as it is currently implemented instead of replacing / with any other ugly stuff.

Reply:
The current behavior is decidedly broken when importing, but we could require the end user to escape their own names. So for example we don't export with the escaped slash, but we recognize and handle it on import. I don't necessarily prefer that route.

Reply:
> I'm in favor of the `\/` solution. It's not perfect, but probably good enough for this niche use case.

Ok sounds good. 
The problem there is that `\/` is not a valid escape sequence. It could be replaced by  `\\/` but then you need more complicated regular expressions when parsing to distinguish a normal `/` from a `\\/`.

Reply:
`\/` in the CSV file, so `\\/` in the C++ string. The split regex would be something like this: `[^\\]/`.